alembic/versions/b6aa2fc6cd42_refresh_usda_views_to_use_api_name.py

```
@@ -0,0 +1,97 @@
+          du.name AS dimension_unit
+        FROM public.observation obs
+        JOIN public.usda_census_record ucr
+          ON obs.record_id = ucr.id::text
+         AND obs.record_type = 'usda_census_record'
+        JOIN public.usda_commodity uc ON ucr.commodity_code = uc.id
+        JOIN public.place p ON ucr.geoid = p.geoid
+        JOIN public.parameter param ON obs.parameter_id = param.id
+        JOIN public.unit u ON obs.unit_id = u.id
+        LEFT JOIN public.dimension_type dt ON obs.dimension_type_id = dt.id
+        LEFT JOIN public.unit du ON obs.dimension_unit_id = du.id
+        """
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)